lms/djangoapps/bulk_email/models.py

```diff
@@ -470,6 +470,26 @@ def __str__(self):
             not_en = ""
         return f"Course '{str(self.course_id)}': Instructor Email {not_en}Enabled"
 
+class DisabledCourse(models.Model):
+    """
+    Disable the bulk email feature for specific courses.
+
+    .. no_pii:
+    """
+
+    course_id = CourseKeyField(max_length=255, db_index=True, unique=True)
+
+    @classmethod
+    def instructor_email_disabled_for_course(cls, course_id):
+        """
+        Returns whether or not email is disabled for the given course id.
+        """
+        try:
+            record = cls.objects.get(course_id=course_id)
+            return record.email_disabled
```

Comment:
It looks like we removed the `emal_disabled` field after an earlier comment. Should we adjust this logic as well?